client/package.json

```diff
@@ -46,27 +46,28 @@
   "bugs": "https://scuttle.atlassian.net/servicedesk/customer/portal/2",
   "homepage": "https://github.com/scpwiki/wikijump#readme",
   "devDependencies": {
+    "@ltd/j-toml": "^1.12.2",
```

Comment:
We already have `toml`, why this as well?

Reply:
This replaces the `toml` package in the dependencies. `toml` can only parse up to (iirc) TOML v0.4 rather than v1.0 like this one.